ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -513,7 +513,9 @@ export default defineComponent({
 
     const defaultHomeOwner: MhrRegistrationHomeOwnerIF = {
       ...props.editHomeOwner,
-      ownerId: props.editHomeOwner?.ownerId || getTransferOrRegistrationHomeOwners().length + 1 || (DEFAULT_OWNER_ID++),
+      ownerId: props.editHomeOwner?.ownerId || props.isMhrTransfer
```

Comment:
We need to track the correct `ownerId` for Transfers vs Registration. Otherwise, we could have owners with the same Id (if deleting the first owner and then adding a new one).

Reply:
Although i said to just remove the parenthesis on the Default on Friday, couldn't the Default be removed entirely? 
ie no ternary and just fallback to the `getTransferOrRegistrationHomeOwners().length `? Wouldn't it resolve the same?

Reply:
I did remove the parenthesis, and it worked for some scenarios, but there are conditions that didn't help. We can't fallback to `getTransferOrRegistrationHomeOwners().length` because in registration, if I add 2 owners and then remove first one and then add the third one - both owners will have `ownerId` of 2.

Reply:
Ah yes, makes sense on the fallback 👍  Thanks for clarifying. 

What scenarios requires the ternary?

Reply:
Basically what I discovered, is that for new registrations, we need `ownerId` to be incremental starting at 1. But for transfers, we need `ownerId` to be incremental from the number of owners in the `getTransferOrRegistrationHomeOwners()` because there could be some existing ones.

Reply:
You bet, always existing ones in Transfers! 

Fair enough, that being said, would make sense to just get getMhrTransferHomeOwners` vs `getTransferOrRegistrationHomeOwners()`